src/content/ui-api/kubectl-gs/template-cluster.md

```diff
@@ -84,16 +95,16 @@ The command to execute is `kubectl gs template cluster`.
 
 It supports the following flags:
 
-- `--provider` - The infrastructure provider (one of: `aws`, `azure`, or `openstack`).
+- `--provider` - The infrastructure provider (one of: `capa`, `aws`, `azure`, or `openstack`).
 - `--name` - Unique name of the cluster. If not provided, a random alphanumeric name will be generated.
 - `--organization` - Name of the organization that will own the cluster. Determines the namespace where resources will be created.
-- `--release` (AWS and Azure only) - Workload cluster release version.
+- `--release` (non CAPI AWS and Azure only) - Workload cluster release version.
   Can be retrieved with `kubectl get releases` for your installation.
 - `--description` (optional) - User-friendly description of the cluster's purpose.
-- `--pods-cidr` (optional) - CIDR applied to the pods. If this isn't provided, the installation default will be applied.
-- `--label` (optional) - workload cluster label in the form of `key=value`. Can be specified multiple times.
-- `--service-priority` (optional) - [Service priority]({{< relref "/advanced/labelling-workload-clusters#service-priority" >}}) of the cluster (one of: `highest`, `medium`, or `lowest`; default: `highest`).
-- `--release-branch` (optional, AWS and Azure only) - The Giant Swarm [releases repository](https://github.com/giantswarm/releases) branch to use to look up the workload cluster release set via the `--release` flag (default: `master`).
+- `--pods-cidr` (optional and non CAPI AWS) - CIDR applied to the pods. If this isn't provided, the installation default will be applied.
+- `--label` (optional and non CAPI AWS) - workload cluster label in the form of `key=value`. Can be specified multiple times.
+- `--service-priority` (optional and non CAPI AWS) - [Service priority]({{< relref "/advanced/labelling-workload-clusters#service-priority" >}}) of the cluster (one of: `highest`, `medium`, or `lowest`; default: `highest`).
+- `--release-branch` (optional, non CAPI AWS and Azure only) - The Giant Swarm [releases repository](https://github.com/giantswarm/releases) branch to use to look up the workload cluster release set via the `--release` flag (default: `master`).
```

Comment:
```suggestion
- `--pods-cidr` (optional and non CAPI AWS) - CIDR applied to the pods. If this isn't provided, the installation default will be applied.
- `--label` (optional and non CAPI AWS) - workload cluster label in the form of `key=value`. Can be specified multiple times.
- `--service-priority` (optional and non CAPI AWS) - [Service priority]({{< relref "/advanced/labelling-workload-clusters#service-priority" >}}) of the cluster (one of: `highest`, `medium`, or `lowest`; default: `highest`).
- `--release-branch` (optional and non CAPI AWS and Azure only) - The Giant Swarm [releases repository](https://github.com/giantswarm/releases) branch to use to look up the workload cluster release set via the `--release` flag (default: `master`).
```

Reply:
I have tried to set labels, pod-cidr and service-priority but I did not see those values applied to the resulting yaml manifests.  
This is why I removed those parameters, for now, until we investigate further.

Reply:
Ah apologies - I have now realized this is about formatting

Reply:
all good :)